doc/Data Plane Code Generation/Sample.md

```diff
@@ -0,0 +1,28 @@
+Samples are where developers could start to learn a library. You should generate samples scaffold by running [quick start tool](AzureSDKCodeGeneration_DataPlane_Quickstart.md#create-starter-package). The samples should match the structure containing the following key features:
```

Comment:
URIs with spaces in them can cause issues in some cases if not properly escaped. Could you at least use dashes in place of spaces or just a shorter name?